pyServer/GuestFishWrapper.py

```diff
@@ -268,283 +276,292 @@ def execute(self, storageUrl):
         lastGoodOperationMinorStep = 1
 
         with open(self.operationOutFilename, "w", newline="\n") as operationOutFile:
```

Comment:
Maybe consider opening both files in the same line via a comma separator to avoid the indentation situation? For example, 

with open("file_1.txt") as f1, open("file_2.txt") as f2:

https://www.kite.com/python/answers/how-to-open-multiple-files-using-with-in-python

Reply:
changed to open two files in a line. moved back the indentation.